python/F3DPythonBindings.cxx

```diff
@@ -14,14 +14,69 @@
 
 namespace py = pybind11;
 
+namespace PYBIND11_NAMESPACE
```

Comment:
possible collision ? should be part of f3d namespace ?

Reply:
Has to be part of `pybind11::detail` for the magic to work I think.

In the docs they do
```cpp
namespace PYBIND11_NAMESPACE { namespace detail {
    template <> struct type_caster<MyClass> {
    // ...
    };
}}
```
It may be possible to inline the namespace rather than enclose
```cpp
template<>
class py::detail::type_caster<f3d::point3_t> { /* ... */ };
```
I'll check if it still works

Reply:
fait enough